docs/engineering/android-app/writing-fhir/fhir_path.mdx

```diff
@@ -300,3 +300,23 @@ Use of other [Value Constraint](https://build.fhir.org/ig/HL7/sdc/behavior.html#
 ]
 }
 ```
+
+## Gotchas
+When working with dates, please note that according to the FHIR spec, if the duration value is specified as a whole number _(e.g. 1 month)_ such as when you write `today + '1 month'`, then when the duration is added or subtracted to a given date(time), the outcome is rounded to the nearest natural calendar division - e.g. Feb. 1 + 1 mo = March 1, not March 2 or 3 (since 1 month is defined in UCUM as 30 days).
+
+**Examples of rounding off to closest calendar division:**
+
+> Feb 1 2023 + (30 days) = March 3, 2023 - According to FHIR Path spec it will be 1st March
+> Feb 2 2023 + (30 days) = March 2, 2023 - According to FHIR Path spec it will be 2nd March
+
+> Feb 1 2024 + (30 days) = March 2, 2024 - According to FHIR Path spec it will be 1st March (leap year)
+> Feb 2 2024 + (30 days) = March 2, 2024 - According to FHIR Path spec it will be 2nd March
+
+> 31st Jan, 2023 + (30 days) = March 2, 2023 - According to FHIR Path it will be 28th Feb
+> 31st Jan, 2024 + (30 days) = March 1, 2024 - According to FHIR Path it will be 29th Feb
+
+> 31st Jan, 2023 + (60 days i.e. '2 month') = April 1, 2023 - According to FHIR Path it will be 31st March
+> 31st Jan, 2024 + (60 days i.e. '2 month') = March 31, 2024 - According to FHIR Path it will be 31st March
```

Comment:
I'm confused by this. Isn't the issue only when you're specifying a larger unit? Like in the FHIR Path spec I would expect Feb 1 + 30 days to be March 3 (or 2nd depending on if it's leap year), but Feb 1 + 1 month to be March 1.

And similarly for the other examples. Although I'm not clear on what Jan 31 + 1 month would be, is that Feb 28 (or 29 in a leap year)?

(Also side not, change this to be consistent, so Jan 31 to match Feb 1 (not 31st Jan)

Reply:
I think I complicated it with all the examples. I think we can remove them since the documentation is kind of clear. Or I could just leave one straightforward example

Reply:
I think I complicated it with all the examples. I think we can remove them since the documentation is kind of clear. Or I could just leave one straightforward example

Reply:
I think I complicated it with all the examples. We can remove them since the documentation is kind of clear. Or I could just leave one straightforward example